src/Microsoft.ML.Data/DataLoadSave/Text/TextLoader.cs

```diff
@@ -520,26 +520,25 @@ public static ColInfo Create(string name, PrimitiveType itemType, Segment[] segs
             }
         }
 
-        private sealed class Bindings : ISchema
+        private sealed class Bindings
         {
+            /// <summary>
+            /// <see cref="Infos"/>[i] stores the i-th column's name type loaded from the input text file.
+            /// </summary>
             public readonly ColInfo[] Infos;
-            public readonly Dictionary<string, int> NameToInfoIndex;
+            /// <summary>
+            /// <see cref="Infos"/>[i] stores the i-th column's metadata, slot names.
+            /// </summary>
             private readonly VBuffer<ReadOnlyMemory<char>>[] _slotNames;
-            // Empty iff either header+ not set in args, or if no header present, or upon load
-            // there was no header stored in the model.
+            /// <summary>
+            /// Empty iff either header+ not set in args, or if no header present, or upon load
```

Comment:
>either header+ not set in args [](start = 26, length = 30)

I might prefer something like an actual reference using a `<see` tag to the `Arguments.HasHeader` field, given that we're now making this an XML comment. Not a big deal though.

Reply:
Now we have
```charp
            /// <summary>
            /// Empty if <see cref="ArgumentsCore.HasHeader"/> is false, no header presents, or upon load
            /// there was no header stored in the model.
            /// </summary>
```

---
In reply to: [247682367](https://github.com/dotnet/machinelearning/pull/2140#discussion_r247682367) [](ancestors = 247682367)